assets/src/edit-story/components/library/panes/media/local/restrictAccessDialog.js

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { Plain } from '../../../../button';
+import Dialog from '../../../../dialog';
+
+function RestrictAccessDialog({ open, onClose }) {
```

Comment:
I'm not sure about the naming of this component (and related properties). It's not a component to "restrict access", but rather a component to inform of "access restriction".

Reply:
MissingPermissionsDialog?

Reply:
`MissingUploadPermissionsDialog` to make it clear which kind of permission?

Reply:
Updated as `MissingUploadPermissionsDialog`